src/app/main/component/ubs/components/ubs-personal-information/ubs-personal-information.component.ts

```diff
@@ -156,8 +173,14 @@ export class UBSPersonalInformationComponent implements OnInit, OnDestroy {
       .subscribe(() => this.findAllAddresses());
   }
 
+  getFormValues(): boolean {
+    return true;
+  }
```

Comment:
what the point of this ?

Reply:
the function "getFormValues" effect on the warning window appearing. It does return a boolean. In this particular case (Step 2), I always need to show the warning window.